qemu/tests/cfg/hpt_max_page_size.cfg

```diff
@@ -0,0 +1,28 @@
+- hpt_max_page_size: install setup image_copy unattended_install
+    type = hpt_max_page_size
+    only ppc64 ppc64le
+    only Linux
+    only virtio_scsi
+    virt_test_type = qemu
+    required_qemu = [4.0.0,)
+    kill_vm = yes
+    start_vm = no
+    pre_command_noncritical = yes
+    setup_hugepages = yes
+    hp_size_cmd = cat /proc/meminfo|grep Huge | grep -E '*HugePages:' | awk '{print $2}'
+    machine_type_extra_params = max-cpu-compat=power8
+    variants:
+        - 64k:
+            machine_type_extra_params += ,cap-hpt-max-page-size=64k
+            variants:
+                - without_men_backing_file:
+                - with_men_backing_file:
+                    extra_params += " -mem-path /mnt/kvm_hugepage"
+        - 16m:
+            machine_type_extra_params += ,cap-hpt-max-page-size=16M
+            no with_men_backing_file
+            variants:
```

Comment:
only one scenario, not need to add variants

Reply:
ok, will change it.